File: src/main/java/wordseeker/gui/table/TableEntry.java

package wordseeker.gui.table;

/**
 * @author rad1kal
 */
public class TableEntry {
    private final String pathToFile;
    private final String foundInFile;

    public TableEntry(String pathToFile, String foundInFile) {
        this.pathToFile = pathToFile;
        this.foundInFile = foundInFile;
    }

    /**
     * @return the pathToFile
     */
    public String getPathToFile() {
        return pathToFile;
    }

    /**
     * @return the foundInFile
     */
    public String getFoundInFile() {
        return foundInFile;
    }

    @Override
    public String toString() {
        return pathToFile.concat(foundInFile);
    }
}

File: src/main/java/wordseeker/gui/table/TableEntry.kt
package wordseeker.gui.table

/**
 * @author rad1kal
 */
class TableEntry(
        /**
         * @return the pathToFile
         */
        val pathToFile: String,
        /**
         * @return the foundInFile
         */
        val foundInFile: String) {

    override fun toString(): String {
        return pathToFile + foundInFile
    }
}
